mfhdf/dumper/hdp_list.c

```diff
@@ -892,39 +892,41 @@ do_list(intn curr_arg, intn argc, char *argv[], intn help)
     int32       an_id     = FAIL; /* annotation interface handle */
     intn        ret_value = SUCCEED;
 
+    /* Do this early, to avoid uninitialized warnings in cleanup code */
+    init_list_opts(&list_opts);
+
```

Comment:
This was initialized after some `goto done` calls, so the compiler complains about the string in list_opts not always being initialized.